ui/vty/src/Ariadne/UI/Vty/Widget/Account.hs

```diff
@@ -243,5 +244,7 @@ performNewAddress = do
 performCopyAddress :: Int -> WidgetEventM AccountWidgetState p ()
 performCopyAddress idx = do
   whenJustM ((^? ix idx) <$> use accountAddressesL) $ \AccountAddress{..} -> do
-    liftIO . setClipboard . toString $ accountAddressHash
-    accountAddressResultL .= AddressCopyResultSuccess accountAddressHash
+    result <- liftIO . handle (\e -> return $ AddressResultError $ show (e :: ClipboardException)) $ do
```

Comment:
You might want to use [`displayException`](https://hackage.haskell.org/package/base-4.11.1.0/docs/Control-Exception.html#v:displayException) instead of calling `show`.

Reply:
Yes, `displayException` should be preferred (even if it's not explicitly defined now, it may be defined later).